CONST_Makefile

```diff
@@ -459,7 +465,6 @@ $(OUTPUT_DIR)/build.js: .build/node_modules.timestamp .build/build.js
 		bootstrap/dist/js/bootstrap.min.js \
 		proj4/dist/proj4.js \
 		d3/d3.min.js \
-		typeahead.js/dist/typeahead.bundle.min.js \
```

Comment:
Yes. But if we add that to c2cgeoportal we should also add typehead as a node dependency to package.json. The problem is that package.json is in the create scaffold while CONST_Makefile is in the update scaffold.